dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/DataSourceServiceTest.java

```diff
@@ -51,46 +55,206 @@
     private DataSourceService dataSourceService;
     @Mock
     private DataSourceMapper dataSourceMapper;
+    @Mock
```

Comment:
You can remove this unused logger.
![image](https://user-images.githubusercontent.com/29545877/90771663-37a45380-e326-11ea-9736-14d2fb4687cf.png)